Makefile

```diff
@@ -1,7 +1,8 @@
 all: build
 .PHONY: all
 
-REPO :=quay.io/scylladb/scylla-operator-images
+REPO :=localhost/scylla-operator-images
```

Comment:
```suggestion
REPO :=quay.io/scylladb/scylla-operator-images
```

Reply:
this is to prevent podman to accidentally download the remote manifest instead of a manifest list and when the remote becomes a manifest list to avoid duplicating the digests for the same architecture

Reply:
I see, lets add a comment since it's not obvious.

Reply:
added comment